test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -35,6 +35,12 @@ def step(context):
     for row in table[1:]:
         _statusChat.send_message(row[0])
         _statusChat.verify_last_message_sent(row[0])
+        
+@Then("user is able to send chat message (without verification)")
+def step(context):
+    table = context.table
+    for row in table[1:]:
+        _statusChat.send_message(row[0])
```

Comment:
I'd change it to accept the message inside the gherkin line, something like 

```suggestion
@Then("user is able to send the chat message |any|")
def step(context, message):
    _statusChat.send_message(message)
```

We can still send multiple messages if we want using examples, but in this case, it makes it way simpler, since we really want to send only one message.